src/test/java/org/broadinstitute/hellbender/tools/spark/sv/KmerTest.java

```diff
@@ -0,0 +1,123 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * Created by tsharpe on 1/12/16.
+ */
+public class KmerTest extends BaseTest
```

Comment:
We already have something named KmerUnitTest (and something called Kmer, as well).  Should I rename to SVKmer or something?
Constructor tests already test toString.  I've renamed the test to make this clear.  (Don't be surprised! A kmer is just a funny kind of number, and numbers and Strings can be a little complicated to convert between.  Go look up Double.toString!)  Besides, reverseComplement is way more complicated, just a lot shorter.